test/cypress/integration/13-contributeFlow-stripePaymentElement.test.js

```diff
@@ -45,6 +42,34 @@ function contributeWithNewUsBankAccount({ email, name } = {}) {
   cy.get('button[data-cy="cf-next-step"]').click();
 }
 
+function contributeNewSEPADebit({ name } = {}) {
+  cy.contains('New payment method').click();
+
+  cy.wait(2000);
+  cy.getStripePaymentElement().within(() => {
+    cy.get('.p-PaymentMethodSelector').then($selector => {
+      if ($selector.find('#sepa_debit-tab').length) {
+        cy.get('#sepa_debit-tab').click();
+      } else {
+        cy.get('.p-AdditionalPaymentMethods-menu').select('sepa_debit');
+      }
+    });
+
+    cy.get('#Field-ibanInput').type('FR1420041010050500013M02606');
+
+    if (name) {
+      cy.get('#Field-nameInput').type(name);
+    }
+
+    cy.get('#Field-countryInput').select('FR');
+    cy.get('#Field-addressLine1Input').type('Street');
+    cy.get('#Field-postalCodeInput').type('01562');
+    cy.get('#Field-localityInput').type('Paris');
+  });
+  cy.get('button[data-cy="cf-next-step"]').click();
+  cy.wait(3000);
+}
+
 describe.skip('Contribute Flow: Stripe Payment Element', () => {
```

Comment:
None of these tests are being executed due to this `skip`.

```
Running:  13-contributeFlow-stripePaymentElement.test.js                                 (4 of 11)


  Contribute Flow: Stripe Payment Element
    Card
      - Guest
      - User
    ACH
      - Guest
      - User
    SEPA
      - Guest
      - User


  0 passing (215ms)
  6 pending
```

```suggestion
describe('Contribute Flow: Stripe Payment Element', () => {
```

Reply:
Yep, these were disabled on a previous PR, I reenabled them here again.